src/System.Globalization/tests/Performance/Perf.CompareInfo.cs

```diff
@@ -0,0 +1,177 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+using Microsoft.Xunit.Performance;
+using System.Collections.Generic;
+
+namespace System.Globalization.Tests
+{
+    public class Perf_CompareInfo
+    {
+        private static string GenerateInputString(char source, int count, char replaceChar, int replacePos)
```

Comment:
this looks identical to the function with the same name below

Reply:
Right, the reason why I copied the function and not moved it to some common place is because we are porting these benchmarks to BDN right now and with that we have a common place for some helper methods. I will address the duplication during the port.